template/main.typ

```diff
@@ -30,7 +30,6 @@
 
   // equation settings
   equate-settings: (breakable: true, sub-numbering: true, number-mode: "label"),
-  numbering-pattern: "(1.1)",
 
```

Comment:
```suggestion
	equation-numbering-pattern: "(1.1)",
	
```
I would like to have all template arguments in the main for now (this is also something we can discuss on)